impress-org/givewp
# Feature: add Mailchimp add-on to the form migration process

Resolves: [GIVE-136](https://stellarwp.atlassian.net/browse/GIVE-136)
## Description

This PR includes a form migration step for the new Mailchimp block migrating from a v2 form to a v3 form. I also removed the commented section of supported addons for mailchimp so that it will display on our list when upgrading.

 Cases when the block should be added.
- While Mailchimp is globally enabled on all forms, if the form is not explicitly disabled from the per form setting migrating should add the block.
- While Mailchimp is not globally enabled if it is explicitly enabled on the per form setting than migrating should add the block.

## Testing Instructions
- Use [New Mailchimp Block 
](https://github.com/impress-org/give-mailchimp/pull/84)
- Create a v2 form -> update the settings -> migrate the form to verify the new settings and block are added if they match one of the above cases. If not verify the block in not added to the new form.
- Verify Mailchimp icon is displayed as a supported addon.

## Pre-review ChecklistT




[GIVE-136]: https://stellarwp.atlassian.net/browse/GIVE-136?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
